### PR TITLE
CoreLocationCLI 2.0.0 (new formula)

### PR DESCRIPTION
**Pull request by adaofeliz**
Command line program to print location information from CoreLocation.
More info: https://github.com/fulldecent/corelocationcli

Notice: We can only test that CoreLocationCLI is installed. CoreLocation framework needs user's interaction to allow its access to the client (CoreLocationCLI). 

CoreLocationCLI_UserInteration

### CODE REVIEW

**Comment by adaofeliz** [comment]
Hi, this is my first Formula. What should I do now? - Cheers!

File: Library/Formula/corelocationcli.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  depends_on :xcode => :build
+
+  def install
+    xcodebuild "-project", "CoreLocationCLI.xcodeproj", "SYMROOT=build", "-sdk", "macosx#{MacOS.version}"
+    bin.install "build/Release/CoreLocationCLI"
+  end
+
+  test do
+    system "(#{bin}/CoreLocationCLI -h; true)"
```
Comment (bfontaine) [inline Library/Formula/corelocationcli.rb]: Please use `system "#{bin}/CoreLocationCLI", "-h"`. If this is expected to exit with a non-zero code like `1`, use `shell_output("#{bin}/CoreLocationCLI -h", 1)` instead.

**Comment by adaofeliz** [comment]
@bfontaine can you check if everything is correct? - Thanks.

**Comment by bfontaine** [comment]
Thank you @adaofeliz for your (first) contribution to Homebrew, we appreciate it! :tada:

**Pull request closed without merging**